tests/filecheck/mlir-conversion/with-mlir/mlir_opt.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: xdsl-opt %s -p mlir-opt{arguments='--cse','--mlir-print-op-generic'} --print-op-generic | filecheck %s
+
+"builtin.module"() ({
+  "func.func"() ({
+    %0 = "arith.constant"() {"value" = 1 : i32} : () -> i32
+    "func.return"() : () -> ()
+  }) {function_type = () -> (), sym_name = "do_nothing"} : () -> ()
+}) : () -> ()
+
+
+// CHECK:         "builtin.module"() ({
+// CHECK-NEXT:      "func.func"() ({
+// CHECK-NEXT:        "func.return"() : () -> ()
+// CHECK-NEXT:      }) {"function_type" = () -> (), "sym_name" = "do_nothing"} : () -> ()
+// CHECK-NEXT:    }) : () -> ()
```

Comment:
Why not use `CHECK-NOT`? Feels like it's intended for exactly this. Plus. this will be more stable to printing changes for all of these ops.

Reply:
I like check-next for the whole thing because then if it prints an error or something unexpected then at least we catch it. I'm not sure how robust check-not would be if mlir-opt prints "error"